src/Vault.sol

```diff
@@ -532,19 +521,6 @@ contract Vault is ERC4626, ERC20Permit, ILiquidationSource, Ownable {
 
   /* ============ Setter Functions ============ */
 
-  /**
-   * @notice Allow a user to disable or activate prize auto claiming.
-   * @dev Auto claim is active by default for all users.
-   * @param _disable Disable or activate auto claim for `msg.sender`
-   * @return bool New auto claim status
-   */
-  function disableAutoClaim(bool _disable) external returns (bool) {
-    autoClaimDisabled[msg.sender] = _disable;
-
-    emit AutoClaimDisabled(msg.sender, _disable);
```

Comment:
The event also needs to be removed: https://github.com/pooltogether/v5-vault/blob/839fa4b9815b270abf01531a96e7df02294c2e7a/src/Vault.sol#L63